sdk/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Could this go at the top level since it applies to both?

Reply:
Yeah good catch - I had already added to the top level, but forgot to remove this one.